plugin.video.drnu/resources/language/resource.language.da_dk/strings.po

```diff
@@ -0,0 +1,157 @@
+# Kodi Media Center language file
```

Comment:
Gotham isn't compatible with the new language directory layout (resource.language.*).
So this file still has to be in the same folder as the strings.xml file was (`plugin.video.drnu/resources/language/Danish/strings.so`).

Reply:
Should the filetype be .so? (typo?) or just be the exact same file and filetype.

Reply:
Copied new strings.po files back to original folder structure

Reply:
`.po` is the correct excetnsion, `.so` is a mistake.
This file should be removed, only `plugin.video.drnu/resources/language/Danish/strings.po` should exist.

Reply:
OK - Removed it in new commit